homeassistant/components/acer_projector/switch.py

```diff
@@ -82,7 +82,7 @@ def __init__(
         )
         self._serial_port = serial_port
         self._attr_name = name
-        self._attr_extra_state_attributes = {
+        self._attributes = {
```

Comment:
I really don't get why you re-introduced this temporary variable again?
You could directly have modified the dictionary in the update() method directly right?
Why re-introduce this?

Reply:
Putting `self._attributes` back to the entity class attributes gives these mypy errors:
```
homeassistant/components/acer_projector/switch.py:131: error: Item "None" of "Optional[Mapping[str, Any]]" has no attribute "__iter__" (not iterable)  [union-attr]
homeassistant/components/acer_projector/switch.py:135: error: Unsupported target for indexed assignment ("Optional[Mapping[str, Any]]")  [index]
```
I'm not sure how to reconcile what your looking for and make mypy happy.

Reply:
aah! That makes sense, yeah that is the case at this point, you are right 👍

Reply:
I think it looks good now